backend/fms_core/migrations/0041_auto_20220816_1946.py

```diff
@@ -0,0 +1,58 @@
+# Generated by Django 3.1 on 2022-08-16 19:46
+
+from django.conf import settings
+from django.db import migrations, models
+import django.db.models.deletion
+import fms_core.schema_validators
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+        ('fms_core', '0040_v3_10_0'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='Dataset',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('created_at', models.DateTimeField(auto_now_add=True, help_text='Date the instance was created.')),
+                ('updated_at', models.DateTimeField(auto_now=True, help_text='Date the instance was modified.')),
+                ('deleted', models.BooleanField(default=False, help_text='Whether this instance has been deleted.')),
+                ('project_name', models.CharField(help_text='Project name.', max_length=200)),
```

Comment:
We will not be getting project names... we would be better off calling this external_project_id (external to make sure we do not confuse with FMS project_id and id so we know this is not a name...).